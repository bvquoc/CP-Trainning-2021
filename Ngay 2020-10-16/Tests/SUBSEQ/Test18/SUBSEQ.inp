600
246 61 89 1 167 305 262 4 269 82 102 41 178 256 292 32 80 264 96 197 23 164 214 57 236 139 313 290 305 170 264 103 56 156 62 183 213 177 49 232 204 264 111 304 326 36 182 181 192 44 110 11 43 92 207 175 132 42 70 176 216 78 176 294 192 147 133 116 174 156 225 302 187 208 248 40 303 301 308 284 150 184 209 214 238 2 318 13 87 132 110 130 224 300 307 6 280 175 26 33 263 8 268 12 66 143 276 20 84 163 197 2 143 26 31 43 143 77 146 109 239 31 289 135 116 309 130 108 107 333 24 326 237 266 269 51 56 57 72 11 145 276 311 9 320 288 4 214 76 252 245 16 217 225 165 263 128 212 218 295 254 60 182 278 219 201 232 258 163 258 49 141 62 242 9 305 57 223 102 100 87 319 139 8 323 37 145 235 126 9 227 318 284 164 324 121 89 258 71 129 209 204 182 328 293 111 200 134 249 220 171 318 310 200 326 48 186 271 252 226 91 101 108 137 32 230 198 107 36 25 164 214 46 104 264 194 136 201 128 280 186 204 232 270 117 176 8 133 12 125 131 228 284 254 1 85 121 122 181 119 25 223 235 87 135 304 257 194 206 275 302 307 174 300 247 258 93 281 330 85 225 254 140 326 82 295 304 230 23 88 98 124 221 136 268 22 324 185 234 120 311 223 317 117 320 35 53 119 245 118 202 37 272 237 89 25 201 283 179 117 45 153 137 115 325 306 257 192 302 42 261 86 103 251 121 277 42 195 228 221 98 275 230 47 294 5 234 134 283 3 68 177 262 188 188 113 154 202 267 117 222 318 244 136 128 187 169 329 203 20 241 315 42 152 3 253 92 99 138 46 276 163 246 236 217 64 187 309 206 249 123 16 209 115 26 289 110 297 162 325 148 55 208 221 121 97 298 145 286 218 7 136 252 227 304 22 100 113 67 23 103 7 51 83 162 274 195 299 35 94 195 59 146 192 204 156 299 104 4 303 10 9 7 38 35 295 148 173 240 103 106 44 285 163 63 262 18 166 201 7 31 324 331 108 5 275 258 46 77 88 74 50 234 239 112 262 96 37 205 226 239 57 15 67 281 61 205 249 1 258 308 27 289 244 326 255 139 231 80 197 118 327 278 240 19 300 259 130 59 239 250 158 64 38 258 251 188 25 227 206 177 55 124 97 221 89 102 112 215 256 112 226 97 108 288 12 3 69 297 284 175 150 78 321 144 175 152 135 101 215 235 78 154 246 151 47 237 101 332 285 24 69 118 324 101 331 43 70 67 6 195 48 202 118 198 303 111 312 128 149 209 11 248 25 239 13 154 127 296 38 331 268 203 146 329 145 222 153 170 97 