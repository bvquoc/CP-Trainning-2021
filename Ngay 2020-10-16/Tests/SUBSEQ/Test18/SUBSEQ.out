179
1 6 7 9 14 15 18 25 27 28 29 31 42 44 45 64 72 75 77 78 79 80 85 87 94 95 97 101 103 107 121 123 126 132 133 134 135 142 143 145 146 150 151 156 160 161 164 168 170 174 176 182 185 188 192 193 195 198 204 205 209 212 213 215 218 219 235 240 243 244 253 254 263 266 270 271 272 274 275 276 278 279 282 284 286 287 295 297 299 301 303 305 309 313 314 318 325 326 327 329 331 334 336 342 345 347 349 353 359 362 363 368 371 372 376 381 383 384 388 390 396 398 400 407 409 413 415 426 428 437 440 446 449 453 456 462 463 466 467 473 474 476 481 485 488 490 491 493 494 495 496 502 503 504 506 507 510 511 515 516 530 535 539 540 544 551 554 557 560 564 566 576 578 583 585 589 591 592 595 